src/k8s/pkg/k8sd/types/cluster_config.go

```diff
@@ -95,7 +95,16 @@ func (c *ClusterConfig) SetDefaults() {
 // ClusterConfigFromBootstrapConfig extracts the cluster config parts from the BootstrapConfig
 // and maps them to a ClusterConfig.
 func ClusterConfigFromBootstrapConfig(b *apiv1.BootstrapConfig) ClusterConfig {
+	authzMode := "Node,RBAC"
+	// Only disable rbac if explicitly set to false during bootstrap
+	if !(*b.EnableRBAC) {
```

Comment:
This will fail if `EnableRBAC` is `nil`. Instead, you can do (first check nil, then dereference):
```suggestion
	if v := b.EnableRBAC; v != nil && !*v {
```

Reply:
Would be useful to start with having a unit test where `EnableRBAC` is nil to see the failure, then fix here

Reply:
ah nice, thx @neoaggelos!